assets/pages/oneoff-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -9,23 +9,16 @@ import ContributionsCheckout from 'components/contributionsCheckout/contribution
 
 import { type PageState as State } from '../oneOffContributionsReducer';
 import OneoffContributionsPayment from './oneoffContributionsPayment';
-import OneoffInlineContributionsPayment from './oneoffInlineContributionsPayment';
 
 
 // ----- State Maps ----- //
 
 function mapStateToProps(state: State) {
-
-  const inlineCardPaymentVariant = state.common.abParticipations.inlineStripeFlowCardPayment;
-
   return {
     amount: state.page.oneoffContrib.amount,
     currencyId: state.common.internationalisation.currencyId,
     country: state.common.internationalisation.countryId,
-    inlineCardPaymentVariant,
-    payment: inlineCardPaymentVariant === 'inline' ?
-      <OneoffInlineContributionsPayment /> :
-      <OneoffContributionsPayment />,
+    payment: <OneoffContributionsPayment />,
```

Comment:
Does this need to live inside the properties, or can it be inlined in the component?

Reply:
I think it does, as we pass either `<OneoffContributionsPayment />`  or  `<RegularContributionsPayment />` to `ContributionsCheckout` depending on whether we are processing a one-off or recurring payment.

Reply:
Yeah I saw that, still I think the properties should dictate the configuration of components, whereas rendering should be under local control. Perhaps an ADT like
```js
type ContributionType = 'oneoff' | 'monthly' | 'annually'
```
could provide that decoupling.